### PR TITLE
Update CEC Inverters for Fall 2023

### LINKED ISSUE #1595: Default inverter no longer in CEC library
Need to check all Detailed PV configuration default inverters and modules and update any that are no longer in the new library.

### PR DESCRIPTION
**Pull request by cpaulgilman**
Update CEC inverter library

* New CEC Inverters.csv file
* Files used for generating library added to samples/CEC Module and Inverter Libraries

Fixes #1595 

**Note** Merge this after merging module update.

### CODE REVIEW

File: deploy/runtime/versions.lk [hunk 1]
```diff
@@ -51,10 +51,38 @@ Some more examples:
...
 
-version_upgrade{'2023.9.19'} = define() {
+version_upgrade{'2023.10.31'} = define() {
 
 	const cfg = config();
 
+	// SAM 2022.11.21 default inverters are not in new CEC inverters library
+	if ( cfg.tech == 'Flat Plate PV' || cfg.tech == 'PV Battery' || cfg.tech == 'High-X Concentrating PV' ) 
+	{
+		old_inverter = value( 'inv_snl_inverter_name' );
```
Comment (mjprilliman) [inline deploy/runtime/versions.lk:61]: When I run the upgrade script old_inverter returns as null: 

image

Then this notice pops up after choosing a new inverter from the list: 

image

Maybe that's normal but wanted to note.

Comment (mjprilliman) [reply deploy/runtime/versions.lk:61]: image

**Review by mjprilliman — APPROVED** [review]
Upgrade worked after updates. Thanks!

**Review by brtietz — APPROVED** [review]
Looks good!

**Pull request merged**